drivers/modem/hl7800.c

```diff
@@ -256,6 +256,7 @@ static const struct mdm_control_pinconfig pinconfig[] = {
 #define MDM_MAX_DATA_LENGTH 1500
 #define MDM_MTU 1500
 #define MDM_MAX_RESP_SIZE 128
+#define MDM_IP_INFO_RESP_SIZE 256
```

Comment:
I am still wondering why you need to use this long string. How many IPv6 addresses you are expecting to store here? The max length of the IPv6 address string is 46 chars (INET6_ADDRSTRLEN) so the length of 256 looks rather long.

Reply:
+CGCONTRDP: response is very long for IPv6. Sierra uses IPv6 address format: `a1.a2.a3.a4.a5.a6.a7.a8.a9.a10.a11.a12.a13.a14.a15.a16`

Reply:
(3 * 16) + 15 for each address in the response. Response has at least IP address, IP gateway, and one DNS address.
Thats at least 189 characters in the response not including the other fields.